gravitee-am-repository/gravitee-am-repository-jdbc/src/main/java/io/gravitee/am/repository/jdbc/management/api/JdbcUserRepository.java

```diff
@@ -842,14 +842,21 @@ public Single<User> update(User item, UpdateActions updateActions) {
 
         Mono<Long> action = update.fetch().rowsUpdated();
 
-
         if (updateActions.updateRequire()) {
-            action = deleteChildEntities(item.getId(), updateActions).then(action);
+            if (item.getId() != null) {
```

Comment:
**remark**: shouldn't be possible to have null ID in this method. If there is a case, then maybe we should throw an error at the beginning of the update method to avoid side effect issue with the update request
` update = addQuotedField(update, USER_COL_ID, item.getId(), String.class);`  wdyt ?

Reply:
valid point. I also added one test